src/pages/billing/Project.js

```diff
@@ -226,61 +230,50 @@ const ProjectDetail = ({ authorizeAndLoadAccounts, billingAccounts, billingProje
     { namespace: billingProject.projectName },
     _.map('workspace', workspaces)
   ), [billingProject, workspaces])
-
-  const maxWorkspacesInChart = 10
   const spendChartOptions = {
     chart: { type: 'bar', style: { fontFamily: 'inherit' } },
     credits: { enabled: false },
-    legend: { enabled: false },
-    series: [{ name: 'Total Cost', data: costPerWorkspace.workspaceCosts }],
+    legend: { reversed: true },
+    plotOptions: { series: { stacking: 'normal' } },
+    series: [
+      { name: 'Compute', data: costPerWorkspace.computeCosts },
+      { name: 'Storage', data: costPerWorkspace.storageCosts },
+      { name: 'Other', data: costPerWorkspace.otherCosts }
+    ],
     title: {
       text: costPerWorkspace.numWorkspaces > maxWorkspacesInChart ? `Top ${maxWorkspacesInChart} Spending Workspaces` : 'Spend By Workspace'
     },
-    tooltip: { valuePrefix: '$' },
+    tooltip: {
```

Comment:
I'm overriding the tooltip so I can use the correct costFormatter (instead of assuming $).